ee/codegen/src/__test__/__snapshots__/chat-message-content.test.ts.snap

```diff
@@ -1,11 +1,11 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
 exports[`ChatMessageContent > ARRAY > should write an array of content correctly 1`] = `
-"ArrayChatMessageContentRequest(
+"ArrayChatMessageContent(
```

Comment:
I'm changing the default behavior to not include the `Request` suffix.

Reply:
[oos] We may need to make sdk changes to support this, will test today